instrumentation/dalli/test/opentelemetry/instrumentation/dalli/instrumentation_test.rb

```diff
@@ -80,11 +80,16 @@
       dalli.set('foo', 'bar')
       exporter.reset
 
-      dalli.instance_variable_get(:@ring).servers.first.stub(:write, ->(_bytes) { raise Dalli::DalliError }) do
+      dalli.instance_variable_get(:@ring).servers.first.stub(:write, ->(_bytes) { raise Dalli::NetworkError }) do
```

Comment:
Is it possible to create an invalid connection instead of stubbing this behavior? E.g. invalid URL or a down host

Reply:
seems reasonable, though it's possible that would still raise a `Dalli::NetworkError` (cant say i've looked too closely at the underlying client binary protocol code) . Tbh, this is blocking a release @robertlaurin would like to kick off tho, so i kinda wouldn't mind kicking the can on this (assuming tests pass 🙃 ).